packages/devextreme/testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -632,6 +632,22 @@ QUnit.test('form.option("onFieldDataChanged", "newHandler") -> check new handler
     assert.equal(onFieldDataChangedStub.callCount, 2, 'new handler is called after editor value is changed');
 });
 
+QUnit.test('onFieldDataChanged must be called once (T1213983)', function(assert) {
```

Comment:
Let's make scenario description more precise plz
```suggestion
QUnit.test('onFieldDataChanged must be called once if new formData contains "length" property (T1213983)', function(assert) {
```

Reply:
[9f919472bf239ce538e466cd6a61e848fbd9df1a]